pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -482,7 +482,7 @@ func (r *ReconcileInfinispan) Reconcile(request reconcile.Request) (reconcile.Re
 				// cluster has a `stopping` wait for all the pods becomes unready
 				reqLogger.Info("Waiting that all the pods become unready")
 				for _, pod := range podList.Items {
-					if pod.Status.ContainerStatuses[0].Ready {
+					if len(pod.Status.ContainerStatuses) > 0 && pod.Status.ContainerStatuses[0].Ready {
```

Comment:
@rigazilla we already have IsPodReady() func in common package. Can we reuse it for all checks or create a new common func as checks pefromce more than once?

Reply:
right @dmvolod! done